recipes/scsd/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "scsd" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/cjkingsbury/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: fe001a4668e4c055b1170cfca20a625e6174fa9e8a1e345cb655558d2f9794a2
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - numpy >=1.16.2
+    - pandas >=0.23.4
+    - scipy >=1.4.1
+    - plotly >=5.1.2
+    - flask >=1.1.2
+    - scikit-learn >=0.24.2
+    - networkx >=2.2
+    - matplotlib-base >=3.2.2
+    - seaborn >=0.11.1
+    - werkzeug >=3.0.3
+
+test:
+  imports:
+    - scsd
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/cjkingsbury/scsd
+  summary: 'SCSD is Python software for the analysis of molecular conformation and deformation in crystal structures'
+  license: ACSL
```

Comment:
I don't think we can redistribute packages under this license.

Reply:
I had a feeling this was going to be the case. Thank you for the quick reply :)